.github/actions/use-ci-nexus-cache/action.yml

```diff
@@ -14,41 +14,41 @@ inputs:
 runs:
   using: composite
   steps:
-  - name: Create ~/.m2/settings.xml with cache configuration
-    shell: bash
-    run: |
-      cat <<EOF > ~/.m2/settings.xml
-      <?xml version="1.0" encoding="UTF-8"?>
-      <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
-        <interactiveMode>false</interactiveMode>
-        <servers>
-          <server>
-            <id>camunda-nexus</id>
-            <username>\${env.NEXUS_USERNAME}</username>
-            <password>\${env.NEXUS_PASSWORD}</password>
-          </server>
-        </servers>
-        <mirrors>
-          <mirror>
-            <id>camunda-nexus</id>
-            <mirrorOf>*</mirrorOf>
-            <name>Camunda Nexus</name>
-            <url>http://repository-ci-camunda-cloud.nexus:8081/content/groups/internal/</url>
-          </mirror>
-        </mirrors>
-      </settings>
-      EOF
+    - name: Create ~/.m2/settings.xml with cache configuration
```

Comment:
Oh, I guess I missed that we don't use it anymore :+1: 

>  is there some missed potential to do so instead of using the github caching?
The idea was that our self-hosted runners wouldn't have to talk to the internet to download most dependencies, saving on egress costs. But now that the runners are in a completely different k8s cluster, they can't even talk to the infra provided Nexus cache.

> should we consider removing it if there isn't

Depends on whether we plan on going back to the infra provided runners anytime soon. But I guess we can recover it from the git history anyway so I'm fine with removing it anyway.